Microsoft.HealthVault/Clients/Deserializers/ThingDeserializer.cs

```diff
@@ -19,11 +21,24 @@ namespace Microsoft.HealthVault.Clients.Deserializers
 {
     internal class ThingDeserializer : IThingDeserializer
     {
+        private readonly Guid _applicationSpecificId = new Guid("a5033c9d-08cf-4204-9bd3-cb412ce39fc0");
```

Comment:
>a5033c9d-08cf-4204-9bd3-cb412ce39fc0 [](start = 65, length = 36)

just curious, what is this guid for? Should you add a comment explaining where it came from?